docs/cases/indicators-of-compromise.asciidoc

```diff
@@ -31,7 +31,7 @@ Install a threat intelligence integration to add indicators to the Indicators pa
 
 . Choose one of the following:
 * From the {security-app} main menu, go to *Intelligence* -> *Indicators* -> *Add Integrations*.
-* From the {kib} main menu, click *Add integrations*. Scroll down the list of integration categories and select *Threat Intelligence* to filter by threat intelligence integrations.
+* From the {kib} main menu, click *Add integrations*. In the search bar, search for *Threat Intelligence* to get a list of threat intelligence integrations.
```

Comment:
Teeny tiny nit: [user input](https://docs.elastic.dev/content-design/emphasis) should be monospace:
```suggestion
* From the {kib} main menu, click *Add integrations*. In the search bar, search for `Threat Intelligence` to get a list of threat intelligence integrations.
```